api/v1alpha1/clusterconfig_types.go

```diff
@@ -271,8 +277,10 @@ func (ExtraAPIServerCertSANs) VariableSchema() clusterv1.VariableSchema {
 	return clusterv1.VariableSchema{
 		OpenAPIV3Schema: clusterv1.JSONSchemaProps{
 			Description: fmt.Sprintf(
-				"Extra Subject Alternative Names for the API Server signing cert. For Docker %v are injected automatically.",
+				//nolint:lll // its a user facing message
+				"Subject Alternative Names for the API Server signing cert. For Docker %v are injected automatically. For Nutanix %v are injected automatically.",
```

Comment:
This will start to become unwieldy when more providers are added. I wonder if we can follow up and provide only a description for the provider this variable schema is a part of? Considering variable schema is ultimately provider specific, this should be possible.